packages/webapp/src/components/Finances/FinancesCarrousel/index.jsx

```diff
@@ -0,0 +1,222 @@
+                <p className={clsx([styles.stat, styles.revenueStat])}>
+                  {currencySymbol}
+                  {totalRevenue}
+                </p>
+              </div>
+              <BsChevronRight />
+            </TextButton>
+            <div className={styles.expenseContainer}>
+              <Text className={styles.expenseTitle}>{t('SALE.FINANCES.TOTAL_EXPENSES')}</Text>
+              <p className={clsx([styles.stat, styles.expenseStat])}>
+                {currencySymbol}
+                {totalExpense}
+              </p>
+            </div>
+          </div>
+          <div className={styles.profitLossSummaryContainer}>
+            <ProfitLossIconLight role="img" aria-label={t('SALE.FINANCES.PROFIT_LOSS')} />
+            <Text className={styles.profitLossTitle}>{t('SALE.FINANCES.PROFIT_LOSS')}</Text>
+            <p className={clsx([styles.stat, styles.profitLossStat])}>
+              {profitLoss < 0 ? '-' : ''}
+              {currencySymbol}
+              {Math.abs(profitLoss)}
+            </p>
+          </div>
+        </div>
+      ),
+      note: t('SALE.FINANCES.CARROUSEL_TEXT.GENERIC'),
+      noteColor: '#72B8B0',
+    },
+    {
+      id: 'expenses',
+      label: t('SALE.FINANCES.EXPENSES'),
+      inactiveBackgroundColor: 'var(--white)',
+      inactiveIcon: <ExpenseIcon aria-label={t('SALE.FINANCES.EXPENSES')} />,
+      activeContent: (
+        <div className={styles.cardContent}>
+          <div className={styles.totalExpensesContainer}>
+            <Semibold className={styles.totalExpensesTitle}>
+              {t('SALE.FINANCES.TOTAL_EXPENSES')}
+            </Semibold>
+            <div className={styles.expensesChartContainer}>
+              <div className={styles.expensesChart}>
+                <Doughnut
+                  data={expenseChartData}
+                  options={{ cutout: '70%', borderWidth: 0 }}
+                  plugins={[ShadowPlugin]}
+                />
+              </div>
+              <div className={styles.totalExpensesStatContainer}>
+                <p className={clsx([styles.stat, styles.totalExpensesStat])}>
+                  {currencySymbol}
+                  {totalExpense}
+                </p>
+              </div>
+            </div>
+          </div>
+          <div className={styles.expensesStatsContainer}>
+            <TextButton
+              className={styles.labourExpensesContainer}
+              onClick={() => history.push('/labour')}
+            >
+              <div>
+                <Text className={styles.labourExpensesTitle}>
+                  {t('SALE.FINANCES.TOTAL_LABOUR')}
+                </Text>
+                <p className={clsx([styles.stat, styles.labourExpensesStat])}>
+                  {currencySymbol}
+                  {labourExpense}
+                </p>
+              </div>
+              <BsChevronRight />
+            </TextButton>
```

Comment:
I'm pretty sure I did this too, but I just realized that `TextButton` is a button (inline element),  so it cannot contain block-level elements.